lib/modules/dosomething/dosomething_api/resources/kudos_resource.inc

```diff
@@ -189,6 +188,29 @@ function _kudos_resource_index($reportbackitems, $count) {
  * @param $term_ids
  */
 function _kudos_resource_create($reportback_item_id, $term_ids, $northstar_id) {
+  if (variable_get('rogue_collection', FALSE)) {
+    global $user;
+    $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
```

Comment:
Something like [this](https://github.com/DoSomething/phoenix/blob/ca42f51d11d54f86fc6244f68b94a693247fca98/lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc#L270-L276).